### ISSUE TITLE
Since v70.1.1 can no longer use Java 8

### ISSUE THREAD

**Issue by chrisbrookes** [sigpwned/chardet4j#33]
Looks like from this version you're using the Java 9 version of the Arrays util class. Using Java 8 I get the following error:
```
Caught: java.lang.NoSuchMethodError: java.util.Arrays.equals([BII[BII)Z
java.lang.NoSuchMethodError: java.util.Arrays.equals([BII[BII)Z
	at com.sigpwned.chardet4j.ByteOrderMark.detect(ByteOrderMark.java:61)
	at com.sigpwned.chardet4j.Chardet.detectCharset(Chardet.java:52)
	at com.sigpwned.chardet4j.Chardet.detectCharset(Chardet.java:44)
	at com.sigpwned.chardet4j.Chardet$detectCharset.call(Unknown Source)
```
No worries if this library is Java 9, but you might want to change the About description which says "Simple, compact charset detection for Java 8+".
Thanks!

**Comment by sigpwned** [comment]
Great flag! Looking into this now.

**Comment by sigpwned** [comment]
Fixed by #35
